src/pages/index.js

```diff
@@ -1,20 +1,23 @@
 import React from "react"
+import {Link} from 'gatsby'
 import 'bootstrap/dist/css/bootstrap.css'
 import {Navbar, Form, Button} from 'react-bootstrap'
 import alpaca from './alpaca-cute.jpeg'
 import "./styles.css"
 
-
 function AlpacaBar() {
     return (
         <div className="alpacaBar">
         <Navbar bg="dark" variant="dark">
-            <Navbar.Brand href="/">Alpacify</Navbar.Brand>
+            <Navbar.Brand href="#home">Alpaca Sponsoring</Navbar.Brand>
```

Comment:
@gregstula I think this was an accident too

Reply:
https://github.com/usenkok/CS361-W20-group33-agile-project/commit/7a3b98f46e4dfe8e0207441a5a4245fd0df8b2ae